src/wasm/kotlin/java/arcs/Entity.kt

```diff
@@ -141,3 +143,42 @@ class StringEncoder(private val sb: StringBuilder = StringBuilder()) {
         sb.append("$prefix${if (flag) "1" else "0"}|")
     }
 }
+
+class TextDelegate {
```

Comment:
Please add KDoc for these classes (in this or a future PR)

Reply:
The delegates went away.